datasafe-encryption/datasafe-encryption-impl/src/main/java/de/adorsys/datasafe/encrypiton/impl/document/CMSDocumentReadService.java

```diff
@@ -55,30 +56,36 @@ public InputStream read(ReadRequest<UserIDAuth, AbsoluteLocation<PrivateResource
      */
     @RequiredArgsConstructor
     private static final class CloseCoordinatingStream extends InputStream {
-
         private final InputStream streamToRead;
         private final List<InputStream> streamsToClose;
-
         @Override
         public int read(byte[] b, int off, int len) throws IOException {
             return streamToRead.read(b, off, len);
         }
-
         @Override
         public int read() throws IOException {
             return streamToRead.read();
         }
-
         @Override
         @SneakyThrows
         public void close() {
-            super.close();
-            streamsToClose.forEach(CloseCoordinatingStream::doClose);
+            List<Exception> exceptions = new ArrayList<>();
```

Comment:
What about CMSDocumentWriteService ? 
I suggest to apply same fix there

Reply:
You said it wont cause problems and actually I was not able to get an open filehandle with it. So no changes ...